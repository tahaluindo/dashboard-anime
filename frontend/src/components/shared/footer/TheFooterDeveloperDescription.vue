<template>
  <section class="py-4 col-span-full">
    <a
      class="btn-sm clamptext-08-11 text-secondary hover:text-white"
      href="https://www.linkedin.com/in/caique-de-camargo-40206756/"
      target="_blank"
    >
      Desenvolvido por Caique de Camargo. Todos os direitos reservados a CMF
      Carnes Nobres.
    </a>
  </section>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss">
</style>