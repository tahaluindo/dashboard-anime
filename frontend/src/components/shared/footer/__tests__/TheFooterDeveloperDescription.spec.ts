import { mount } from "@vue/test-utils";
import TheFooterDeveloperDescription from "../TheFooterDeveloperDescription.vue";

describe("TheFooterDeveloperDescription component test", () => {
  it("should render the default component", () => {
    const wrapper = mount(TheFooterDeveloperDescription);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
