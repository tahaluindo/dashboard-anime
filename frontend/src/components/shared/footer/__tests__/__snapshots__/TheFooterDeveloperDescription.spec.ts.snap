// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooterDeveloperDescription component test should render the default component 1`] = `"<section class=\\"py-4 col-span-full\\"><a class=\\"btn-sm clamptext-08-11 text-secondary hover:text-white\\" href=\\"https://www.linkedin.com/in/caique-de-camargo-40206756/\\" target=\\"_blank\\"> Desenvolvido por Caique de Camargo. Todos os direitos reservados a CMF Carnes Nobres. </a></section>"`;
